src/Microsoft.AspNet.SignalR.Client/Transports/ServerSentEventsTransport.cs

```diff
@@ -53,7 +53,9 @@ protected override void OnStartFailed()
             // if the transport failed to start we want to stop it silently.
             _stop = true;
 
-            _request.Abort();
+            var request = _request;
+            if (request != null)
```

Comment:
Issue https://github.com/SignalR/SignalR/issues/3376 shows that it can be null, but I don't know when.